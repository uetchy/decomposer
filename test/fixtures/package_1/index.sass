@import styles

body
  font-family: $ff-gothic
