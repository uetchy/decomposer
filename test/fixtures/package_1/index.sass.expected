@import custom_components/styles/index.sass

body
  font-family: $ff-gothic
